tests/hilti/optimization/peephole-error-push-pop.hlt

```diff
@@ -0,0 +1,19 @@
+# @TEST-EXEC: hiltic -V -p %INPUT >opt.hlt
```

Comment:
Could you add the `optimizer` debug log to the test baseline so we are reminded to potentially simplify/remove the peephole optimizer once this is provided by a flow-sensitive optimizer pass via #1690?

Reply:
optimizer baselines are pretty noisy with existing tests already, prefer not to add more just as a "reminder"; sounds more like a ticket.

Reply:
Added as an item to #1690.